Automattic/wp-calypso
# Jetpack Backup: update daily backup card once a backup is completed

Resolves https://github.com/Automattic/jetpack-backup-team/issues/159
Inspired by #89638

If a user starts a new backup on demand, or if they open the backup page when a backup is in progress, when it completes we show a `Backup just completed` view, telling them the backup will be ready in a few minutes. If the user wants to view the files or restore to this backup, they will need to refresh the page.

The idea of this PR is to update the backup status as soon as possible, instead of let them be in the `Backup just completed` state forever. This approach involves to refetch the backups from the Activity Log every 5 seconds once the backup is completed, so it will automatically update when a new backup arrived.

## Proposed Changes

* Pass `refetch` from `useRewindableActivityLogQuery` upstream to allow manual refresh.
* Refetch backups from Activity Log every 5 seconds after a backup is completed until we got the new backup.
* Updated the component import and all references from `MostRecentStatus` to `DailyBackupStatus` for improved readability and maintainability.

## Demo

https://github.com/Automattic/wp-calypso/assets/1488641/9a29e021-5f8d-4ddf-addb-a8bffa84c0d2

## Testing Instructions



* Spin up a Calypso or Jetpack Cloud live branch
* Select a site with Jetpack VaultPress Backup
* Click on `Back up now`
* Wait until it finished. It should display a `Backup just completed` view, and after some time it will automatically update into a `Latest backup` view.

## Pre-merge Checklist
